src/App.tsx

```diff
@@ -18,30 +16,23 @@ const i18nInstance = initializeI18n({
 });
 
 const App = () => {
-    const [ShouldRedirectToDerivApp, isGBLoaded] = useGrowthbookGetFeatureValue({
-        featureFlag: 'redirect_to_deriv_app_p2p',
-    });
-    const [isOAuth2Enabled] = useOAuth2Enabled();
```

Comment:
This growthbook is not used anymore and is activated for all
<img width="1368" alt="Screenshot 2025-04-08 at 6 01 42 PM" src="https://github.com/user-attachments/assets/b8bb334a-6804-4863-a5df-260cac730a10" />